### PR TITLE
docs: Add readme file for evenlength subject

### PR DESCRIPTION
**Pull request by eslopfer**
Add readme file for evenlength subject.

### CODE REVIEW

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/evenlength/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Here is a possible program to test your function:
+
+```go
+package main
+import (
+    "piscine"
+    "fmt"
+)
+func main() {
+    l := piscine.EvenLength([]string{"Hi", "Hola", "Ola", "Ciao", "Salut", "Hallo"})
```
Comment (MSilva95) [inline subjects/evenlength/README.md:25]: Remove the piscine imports and the variable, just print it

File: subjects/evenlength/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## evenlength
+
+### Instructions
+
+Write a function that receives a slice of strings and returns a new slice with the strings in which the length is even.
```
Comment (MSilva95) [inline subjects/evenlength/README.md:5]: explain what happens when its an empty slice or when there is none even strings

File: subjects/evenlength/README.md (hunk 1 above)
Comment (eslopfer) [reply subjects/evenlength/README.md:25]: Thank you.

File: subjects/evenlength/README.md (hunk 2 above)
Comment (eslopfer) [reply subjects/evenlength/README.md:5]: Thanks. Done.

**Pull request merged**